client/components/happiness-support/README.md

```diff
@@ -10,19 +9,19 @@ import React from 'react';
 import HappinessSupport from 'components/happiness-support';
 
 export default () => {
-    return (
-        <HappinessSupport
-            isJetpack
-            liveChatButtonEventName="calypso_chat_button_clicked"
-            showLiveChatButton
-        />
-    );
+	return (
+		<HappinessSupport
+			isJetpack
+			liveChatButtonEventName="calypso_chat_button_clicked"
+			showLiveChatButton
+		/>
+	);
```

Comment:
I wonder why this was re-formatted when the option for JS was off?

Reply:
I think it is because changes related to alignment (spaces to tabs) are applied everywhere, even inside blocks? Not 100% sure tbh.